src/util/index.js

```diff
@@ -25,6 +25,15 @@ var util = {
     /** {@link Version} */
     Version: require("./Version"),
 
+    /** expose some dependencies for hooks */
```

Comment:
Yes, they are node modules. But not build-in modules, need to be installed. Hooks are executed from temporary dir in which these modules can't be required.